### ISSUE TITLE
Webspoof still safe to use?

### ISSUE THREAD

**Issue by hustenloeser1** [iam4x/pokemongo-webspoof#390]
Hello,

I would just like to know if using this code is still safe, as many people got banned permanently now for using gps fake. 

I just want to go around my city by the subway option, so no big distances.

**Comment by ehad16** [comment]
No, it's easily detectable on ingress after some testing in it's current state.  It's never been safe.  The only reason this rep isn't lit up with bans yet, is because Niantic is going after botting first.  You can pretty much kiss your account goodbye in a few weeks.

**Comment by CreatingMazes** [comment]
It is safe if you use it wisely :)
